safekeeper/client/src/mgmt_api.rs

```diff
@@ -137,7 +137,7 @@ impl Client {
     }
 
     pub async fn utilization(&self) -> Result<SafekeeperUtilization> {
-        let uri = format!("{}/v1/utilization/", self.mgmt_api_endpoint);
+        let uri = format!("{}/v1/utilization", self.mgmt_api_endpoint);
```

Comment:
was this the cause of failure?

Reply:
I don't think so:

1. there is log entries where heartbeats work, so it's not a "doesn't work anywhere" scenario which would be explained by a wrong URL.
2. url libraries in general should handle trailing and double slashes etc.